pkg/storage/export/export/export.go

```diff
@@ -1409,3 +1410,12 @@ func (ctrl *VMExportController) createExportHttpDvFromPVC(namespace, name string
 	}
 	return nil
 }
+
+func (ctrl *VMExportController) getExportVolumeName(pvc *corev1.PersistentVolumeClaim, vmExport *exportv1.VirtualMachineExport) string {
```

Comment:
shouldn't you add this function to the file it is being used at i.e: pkg/storage/export/export/links.go

Reply:
If we are passing the function as argument I think it makes more sense to have it on `export.go`. If we go back to the first commit, I agree with you.